### ISSUE TITLE
Table translation issues

### ISSUE THREAD

**Issue by mihob** [verbb/Events#163]
### Describe the bug

There are a few problems with the translation of strings in different tables, at least for German. I'm not sure if this is a problem with craft or the events plugin.

For example, the ticket type table within an event:
<img width="1066" alt="grafik" src="https://github.com/user-attachments/assets/a22faf6b-632d-4ec7-84f2-253f694439ef">

The labeling of the pagination is not correct, nor is the labeling of the button. Capitalization is not correct here and the space should not be removed.

Is it possible to translate the strings yourself?

### Steps to reproduce

1. 
2. 

### Craft CMS version

5.5.0.1

### Plugin version

3.0.1

### Multi-site?

YES

**Comment by engram-design** [comment]
We don't have bundled translations for anything other than English, so by all accounts, this should read as "New Ticket Type" and "10 ticket types".

Here's what that looks like when switching my CP language to German:

<img width="1006" alt="image" src="https://github.com/user-attachments/assets/9f0d01b6-debc-4410-b275-68ae6e2478f8">

Are you translating those strings yourself?

**Comment by mihob** [comment]
Yes, I overwrite your translations strings in an own translation file under `translations/events.php`. I probably should have mentioned that.

The question is whether you can translate the complete strings in both cases. Inserting the translations for the term there does not necessarily work for languages other than English.

**Comment by engram-design** [comment]
So it's essentially two strings, the `app` category is `'New {type}'` and that's pretty much already translated by Craft, I believe. Then in the `events` category, it's just `Ticket Type` and  `Ticket Types`, which is transposed to `lowerDisplayName` and `pluralLowerDisplayName`.

Can you let me know what your translation string are for that, and what your expected results would be?

**Comment by mihob** [comment]
Hmm, I had a look at the translations of craft in the app category and how it handles entries.

It has its own translations for lowerDisplayName and pluralLowerDisplayName.

```
    public static function displayName(): string
    {
        return Craft::t('app', 'Entry');
    }

    /**
     * @inheritdoc
     */
    public static function lowerDisplayName(): string
    {
        return Craft::t('app', 'entry');
    }

    /**
     * @inheritdoc
     */
    public static function pluralDisplayName(): string
    {
        return Craft::t('app', 'Entries');
    }

    /**
     * @inheritdoc
     */
    public static function pluralLowerDisplayName(): string
    {
        return Craft::t('app', 'entries');
    }
```


This is better suited for German translations as the words would not be lower case here.

**Comment by mihob** [comment]
But to be honest, this is not so important and rather nice to have. You would probably have to adapt all elements of the plugin accordingly.

**Comment by engram-design** [comment]
So we just supply a `displayName()` and `pluralDisplayName()` (see https://github.com/verbb/events/blob/890819719b392a2572e47fb627d027fcf8c21e9e/src/elements/TicketType.php#L39-L47) because according to the base [Element](https://github.com/craftcms/cms/blob/eea8d8bd426b20b808547d448edbccbedf082ed6/src/base/Element.php#L786-L813) class, the `lowerDisplayName()` and `pluralLowerDisplayName()` functions just take the other functions and lowercase them. I would have _thought_ falling back to that be fine.

But, all too happy to add those extra functions if that's going to help localize?

**Comment by mihob** [comment]
Well, it would work for german localization and this is the way craft does it

**Comment by engram-design** [comment]
Updated for the next release. To get this early, run `composer require verbb/events:"dev-craft-5 as 3.0.2"`.

**Comment by engram-design** [comment]
Fixed in [3.0.3](https://github.com/verbb/events/releases/tag/3.0.3)
